### PR TITLE
More validation on lon lat parameters

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Comment by sbrunner** [comment]
See: https://sentry.camptocamp.com/geo/geomapfish-demo/issues/148186/?environment=2.4

File: geoportal/c2cgeoportal_geoportal/views/raster.py [hunk 1]
```diff
@@ -60,24 +61,26 @@ def handle(event: InvalidateCacheEvent):  # pylint: disable=unused-variable
                 v.close()
             Raster.data = {}
 
-    @view_config(route_name="raster", renderer="decimaljson")
-    def raster(self):
-        if 'lon' not in self.request.params:
-            raise HTTPBadRequest("'lon' should be in the query string parameters")
-        if 'lat' not in self.request.params:
-            raise HTTPBadRequest("'lat' should be in the query string parameters")
+    def _get_required_finte_float_param(self, name: str) -> int:
```
Comment (fredj) [inline geoportal/c2cgeoportal_geoportal/views/raster.py]: `finite`instead of `finte` ?

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/views/raster.py]: Thanks, done :-)

File: geoportal/c2cgeoportal_geoportal/views/raster.py [hunk 2]
```diff
@@ -60,24 +61,26 @@ def handle(event: InvalidateCacheEvent):  # pylint: disable=unused-variable
                 v.close()
             Raster.data = {}
 
-    @view_config(route_name="raster", renderer="decimaljson")
-    def raster(self):
-        if 'lon' not in self.request.params:
-            raise HTTPBadRequest("'lon' should be in the query string parameters")
-        if 'lat' not in self.request.params:
-            raise HTTPBadRequest("'lat' should be in the query string parameters")
+    def _get_required_finite_float_param(self, name: str) -> int:
```
Comment (fredj) [inline geoportal/c2cgeoportal_geoportal/views/raster.py]:
```suggestion
    def _get_required_finite_float_param(self, name: str) -> float:
```
?

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/views/raster.py]: Thanks :-)

**Pull request merged**